src/gps_to_pose_conversion_node.cpp

```diff
@@ -34,6 +35,9 @@ double g_covariance_orientation_x;
 double g_covariance_orientation_y;
 double g_covariance_orientation_z;
 std::string g_frame_id;
+std::string g_tf_child_frame_id;
+
+tf::TransformBroadcaster *tf_broadcaster;
```

Comment:
You can't create directly the global object before node handler has been initialized. You'd get the following error: 
`[FATAL] [1486111146.106631943]: You must call ros::init() before creating the first NodeHandle`
That's why I need a raw pointer